packages/shared/src/store/sync.ts

```diff
@@ -113,12 +113,10 @@ function checkForNewlyJoined({
 }
 
 export const syncLatestPosts = async (reporter?: ErrorReporter) => {
-  const result = await syncQueue.add('latest-posts', async () =>
-    Promise.all([
-      api.getLatestPosts({ type: 'channels' }),
-      api.getLatestPosts({ type: 'chats' }),
-    ])
-  );
+  const result = await Promise.all([
```

Comment:
Was sync queue causing an issue?

Reply:
I want it to run immediately when called and didn't want to have to worry about the queue getting cleared.

I think until we move things over to a priority based system, it's gonna be simpler to avoid the queue for anything mission critical.

Reply:
How about this -- we set the concurrency for the `syncQueue` to `Infinity` so that it runs everything immediately. That way we can maintain the bottleneck + logging for when we move to priority queue without adding overhead.

Reply:
Actually whatever, this is gonna get complicated. Remove it + we'll hash it out later lol.

Reply:
haha ya, lets chat tomorrow. I think as we start mapping out the background sync stuff, updating the queue to work the way we want could be a reasonable first step